orttraining/orttraining/training_ops/cuda/optimizer/lamb.h

```diff
@@ -31,6 +32,7 @@ class LambOptimizer final : public CudaKernel {
   std::vector<float> epsilon_;
   float ratio_min_;
   float ratio_max_;
+  int64_t do_bias_correction_;
```

Comment:
>int64_t [](start = 2, length = 7)

when it come to kernel private member, I feel it's better to store as bool

Reply:
Sure.